From 6f72ebe170db5364287f3634359739d3f0c1c987 Mon Sep 17 00:00:00 2001
From: Robert Church <robert.church@windriver.com>
Date: Tue, 15 Jan 2019 03:39:27 -0500
Subject: [PATCH 8/8] Stein: add log_config_append to neutron-etc

neutron-sanity-check command triggers privsep code and produces the
following:

WARNING oslo.privsep.daemon [-] privsep log:
oslo_log.log.LogConfigError: Error loading logging config
/etc/neutron/logging.conf: [Errno 2] No such file or directory:
'/etc/neutron/logging.conf'

This will allow ovs-agent to successfully deploy

Signed-off-by: Robert Church <robert.church@windriver.com>
---
 neutron/templates/daemonset-ovs-agent.yaml | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/neutron/templates/daemonset-ovs-agent.yaml b/neutron/templates/daemonset-ovs-agent.yaml
index 34aba253..af765bd7 100644
--- a/neutron/templates/daemonset-ovs-agent.yaml
+++ b/neutron/templates/daemonset-ovs-agent.yaml
@@ -95,6 +95,10 @@ spec:
               subPath: neutron.conf
               readOnly: true
             - name: neutron-etc
+              mountPath: {{ .Values.conf.neutron.DEFAULT.log_config_append }}
+              subPath: {{ base .Values.conf.neutron.DEFAULT.log_config_append }}
+              readOnly: true
+            - name: neutron-etc
               mountPath: /etc/neutron/plugins/ml2/ml2_conf.ini
               subPath: ml2_conf.ini
               readOnly: true
-- 
2.16.5

